consensus/service/src/config/network.rs

```diff
@@ -26,22 +26,26 @@ pub struct NetworkConfig {
 
 impl NetworkConfig {
     /// Get the network configuration by loading the network.toml/json file.
-    pub fn load_from_path(path: impl AsRef<Path>, peer_responder_id: &ResponderId) -> Self {
+    pub fn load_from_path(
+        path: impl AsRef<Path>,
+        peer_responder_id: &ResponderId,
+    ) -> Result<Self, String> {
```

Comment:
This function was changed to return a `Result` so that all the panicing that previous took place here could be consolidated into the caller site.